main.go

```diff
@@ -37,11 +39,23 @@ type Config struct {
 	SentryDsn              string `required:"false" split_words:"true"`
 }
 
+const ConfigFilePath = "/usr/local/etc/draupnir.toml"
```

Comment:
`/usr/local/etc` is very rarely used in my experience. Any reason for that over `/etc/draupnir/config.toml`?

Reply:
I thought it would fit nicer with a tool that is installed in `/usr/local/bin`...

Reply:
While people use `/usr/local/bin`, they tend not to use `/usr/local/etc`. It's not something I can explain, but `/usr/local/etc` is just a deadzone.

To confirm my suspicion, I just ran a search across our infrastructure. There's not a single file in `/usr/local/etc` on any of our machines in any environment, while `/usr/local/bin` is heavily used.

Reply:
Cool - have gone with `/etc/draupnir/config.toml` then